content/en/docs/reference/kubernetes-api/workload-resources/horizontal-pod-autoscaler-v2.md

```diff
@@ -1,11 +1,11 @@
 ---
 api_metadata:
-  apiVersion: "autoscaling/v2beta2"
```

Comment:
Will this change mean that the docs for HorizontalPodAutoscaler `autoscaling/v2beta2` disappear? This might surprise readers given that lots of tools do use this API at `autoscaling/v2beta2`.

Reply:
Yes, this change will make it disappear. I will check if we can have both v2beta2 and v2 in documentation

Reply:
@sftim I've updated the doc to display v1, v2beta2 and v2 for HorizontalPodautoscaler:
https://deploy-preview-29668--kubernetes-io-vnext-staging.netlify.app/docs/reference/kubernetes-api/workload-resources/

The problem is that versions are not displayed in the titles for non beta versions, and thus it is difficult to distinguish between v1 and v2

What would we like to do?
1. have only v2 and v2beta2?
2. always display the versions in the title (this will display v1 for a lot of resources)

(2) will need to update the program building the documentation, where (1) only needs to modify the configuration in this repository.

Reply:
Could we countenance a manual tweak, post build, for just the HPA APIs to show v1 and v2 explicitly, possibly followed by a logic tweak to show the version any time there are multiple stable APIs?

If we go for the logic tweak, SIG Docs can look for someone to pick up that work (ie: it doesn't have to be you @feloy - you already help a lot here!)

Reply:
We should definitely aim to leave in docs for HPA v1; until the v1.23 release it's the API we officially recommend (v2beta2 is, of course, only beta). The day after that release folks would still expect to see docs for that v1 API available.

Reply:
What do you prefer @feloy ?

Reply:
> Could we countenance a manual tweak, post build, for just the HPA APIs to show v1 and v2 explicitly, possibly followed by a logic tweak to show the version any time there are multiple stable APIs?

> If we go for the logic tweak, SIG Docs can look for someone to pick up that work (ie: it doesn't have to be you @feloy - you already help a lot here!)

@sftim, Yes, it would be nice to have someone make some manual tweak after build with the necessary changes, so I can work on the logic one once I have a precise example of what to do.